pyiron_atomistics/atomistics/structure/factories/aimsgb.py

```diff
@@ -19,75 +21,105 @@
 
 class AimsgbFactory:
     @staticmethod
-    def info(axis, max_sigma):
+    def info(axis,
+             max_sigma=15,
+             specific=False,
+             table_view=False
+            ):
         """
-        Provides a list of possible GB structures for a given rotational axis and upto the given maximum sigma value.
-
+        A dictionary with information including possible sigma, CSL matrix, GB plane,
+        rotation angle and rotation matrix. 
         Args:
-            axis : Rotational axis for the GB you want to construct (for example, axis=[1,0,0])
-            max_sigma (int) : The maximum value of sigma upto which you want to consider for your
-            GB (for example, sigma=5)
+            axis ([u, v, w]): Rotation axis.
+            max_sigma (int): The largest sigma value. Dafult value is 15.
+            specific (bool): Whether collecting information for a specific sigma.
+                Dafult value is False.
+            table_view (bool): Display output in a tabuler format. Default value is True.
 
         Returns:
-            A list of possible GB structures in the format:
+            A aimsgb.GBInformation object or display a table depending on 
+            the value of table_view parameter.
 
-            {sigma value: {'theta': [theta value],
-                'plane': the GB planes")
-                'rot_matrix': array([the rotational matrix]),
-                'csl': [array([the csl matrix])]}}
-
-        To construct the grain boundary select a GB plane and sigma value from the list and pass it to the
-        GBBuilder.gb_build() function along with the rotational axis and initial bulk structure.
+        To construct the grain boundary select a GB plane and sigma value from 
+        the table and pass it to the GBBuilder.build() function along with 
+        the rotational axis and initial bulk structure.
         """
-        return get_grainboundary_info(axis=axis, max_sigma=max_sigma)
+        if table_view:
+            return print(GBInformation(axis, max_sigma, specific).__str__())
```

Comment:
`return print(...)` looks super weird. Is it really not enough to just return `__str__()`? or `print()` without the return?

Reply:
@liamhuber Thanks a lot for the feedback. I will implement the changes you suggested.